src/components/Footer/Footer.tsx

```diff
@@ -23,6 +23,7 @@ type FooterProps = LangSwitchProps & {
   preLoginLinks: PreLoginFooterLinksType;
   legalInfo: JSX.Element | Array<JSX.Element>;
   onExit?: (href: string, linkType: LinkType) => void;
+  productsJsonUrl: string;
```

Comment:
I think that productsJsonUrl should be optional, as optional was the product list before of this requirement.
Moreover, you have to ask also the onProductsJsonFetchError prop in order to provide it to the FooterPreLogin